README.rst

```diff
@@ -40,12 +40,22 @@ iati-registry-record-schema.xsd An application-specific extension
 tests/                          Unit tests for the schemas.
 =============================== ========================================
 
+Dev Installation
+----------------
+
+.. code-block:: bash
+  # Create and start a virtual environment
+  python3 -m venv pyenv
+  source pyenv/bin/activate
+
+  # Install Python package dependencies
+  pip install -r requirements-dev.txt
+
 The tests include a series of short XML documents that should pass or
-file when parsed against the schemas in this distribution.  The shell
-scripts for running the tests rely on a Unix environment with the bash
-shell and the xmllint utility, but the test documents will work with
-any schema-aware XML parser. We will add unit tests as schema
-development continues.
+fail when parsed against the schemas in this distribution.  We use pytest
```

Comment:
Text shouldn't be wrapped mid-sentence - it's hard to maintain and leads to confusing diffs.

Reply:
bd600b5